witness/src/Dscp/CommonCLI.hs

```diff
@@ -38,11 +38,25 @@ import Dscp.Util
 import Dscp.Web (NetworkAddress (..), ServerParams (..), parseNetAddr)
 import Paths_disciplina_witness (version)
 
+{- |
```

Comment:
Minor: I also want to start using such notes and checked which format are you going to keep them in.

And I couldn't ensure whether is such haddock syntax valid, and it definitely is, however, it looks like
![screenshot from 2018-10-03 18-40-13](https://user-images.githubusercontent.com/5394217/46422037-2656fb80-c73c-11e8-82bc-185d9f32a195.png)
when I insert this block above `traceShow` with its comment. So probably `|` would be an extra here.